src/components/StatsGroup/statsGroup.stories.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright Kyndryl, Inc. 2023
+ */
+
+import { html } from 'lit';
+import './statsGroup';
+
+export default {
+  title: 'Components/Stats Group',
+  component: `kd-statsgroup`,
+};
+
+const args = {};
+
+export const Template = {
+  args,
+  render: () => {
+    return html`
+      <kd-statsgroup>
+        <div slot="text-one">
+          <div class="title">
+            <svg
```

Comment:
We need to check with design. These probably shouldn't be svgs. SVGs lack accessibility and number formats change based on nation.